### ISSUE TITLE
Train ML model prototype with historical data

### ISSUE THREAD

**Issue by Tims777** [amosproj/amos2023ws06-sales-lead-qualifier#154]
* Take enriched data.
* Split it into two parts (training and validation).
* Train a ML model with first part.
* Show that it actually learnt something by letting it run on the second half.
* Data columns to use: Categories (XS, S, M, ...) and some correlated other columns from our enriched data

### Notes
* Try to work on  / experiment with different model types in parallel if possible.
* Subsampling / oversampling might be worth a try.
* Start with simple models.

### Acceptance Criteria
* Have a trained model that is able to do some predictions.
  * Use F1-score (or similar) to evaluate model efficiency.

**Comment by ultiwinter** [comment]
We also need to make sure we use the AWS hardware for this, we could leverage  SageMaker's powerful remote hardware instead of our local (suboptimal) CPUs, considering such large data. This could be done on AWS's notebooks directly, but I recommend we find a way of implementing it using our coding framework on github, such that it will be easier and quicker to access and the commits to be visible for AMOS team.

**Comment by felix-zailskas** [comment]
Good point. Perhaps there is a possibility to schedule training jobs on SageMaker via some python API. This way we could store all models on AWS and work with them there while still having the logic in our repo.

**Comment by felix-zailskas** [comment]
https://sagemaker.readthedocs.io/en/stable/
